tests/ProxyManagerTestAsset/IterableTypeHintClass.php

```diff
@@ -28,12 +28,7 @@
  */
 class IterableTypeHintClass
 {
-    /**
-     * @param iterable $parameter
-     *
-     * @return iterable
```

Comment:
@ondrejmirtes PHPStan was complaining here:

```
 ------ ----------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/ProxyManagerTestAsset/IterableTypeHintClass.php                                                                              
 ------ ----------------------------------------------------------------------------------------------------------------------------------- 
  38     Method ProxyManagerTestAsset\IterableTypeHintClass::iterableTypeHintMethod() should return mixed[] but returns iterable(mixed[]).  
 ------ -----------------------------------------------------------------------------------------------------------------------------------
```

I removed the redundant docblock, but it might be an issue.

Reply:
Correct would be sth like this:

```php
/**
 * @param string[] $paremeter
 * @return string[]
 */
```

Reply:
0.9 will work with iterables and generally with all types better.